cmd/influxd/run.go

```diff
@@ -642,19 +642,8 @@ func (cmd *RunCommand) openServer(joinURLs []url.URL) *influxdb.Server {
 	// Give brokers time to elect a leader if entire cluster is being restarted.
 	time.Sleep(1 * time.Second)
 
-	if s.ID() == 0 && s.Index() == 0 {
```

Comment:
So why wasn't this condition correct? Why was also requiring `Index()` to be 0 wrong?